batch/batch/file_store.py

```diff
@@ -39,6 +45,15 @@ async def write_log_file(self, format_version, batch_id, job_id, attempt_id, tas
         url = self.log_path(format_version, batch_id, job_id, attempt_id, task)
         await self.fs.write(url, data.encode('utf-8'))
 
+    async def read_resource_usage_file(self, format_version, batch_id, job_id, attempt_id, task):
+        url = self.resource_usage_path(format_version, batch_id, job_id, attempt_id, task)
+        data = await self.fs.read(url)
+        return ResourceUsageMonitor.decode_to_df(data).to_json()
```

Comment:
I think we should just keep the data in a DataFrame. The output of this function goes straight into `plot_resource_usage`, right? `plot_resource_usage` needs to convert it back to a data frame anyway so that it can send the data into plotly.

Reply:
The multipart is to essentially send 3 binary files together. Same as if you were downloading multiple images at the same time. This way we work with the raw binary and don't use the `to_json()` anywhere.

Reply:
If I can't get the multipart to work easily, then I think I should just make 3 separate requests to get the data.

Reply:
Hmm. I still don't follow. Who is making a multipart request to whom?

Regarding, `to_json`, I don't think we need to convert the data frame to json. We use the data frame to construct a plotly plot, right? And then the plotly plot will be converted to json, but not the data frame itself.